helpme/templates/helpme/team_detail.html

```diff
@@ -7,37 +7,51 @@ <h1>Manage {{team.name}} Team</h1>
 <form method="POST">
     {% csrf_token %}
     {{ form|crispy }}
-    <input type="submit" value="Update" class="mt-3">
+    <input type="submit" value="Update" class="btn helpme-button py-2 px-4 mt-3">
```

Comment:
We need to be cautious about using any custom CSS as we'll likely not be able to include it in our templates.  It is also less portable this way.  Better to use basic bootstrap.